lib/rubocop/cop/rspec/multiple_describes.rb

```diff
@@ -36,6 +101,59 @@ def on_top_level_group(node)
 
           add_offense(node.send_node)
         end
+
+        def on_send(node)
+          return unless style == :splitting
+
+          return unless describe?(node)
+
+          return if includes_common_codes?(node)
+
+          block_node = node.parent
```

Comment:
The parent of a `send` node is not always a block node:
```ruby
def a
  x(1)
end
```
```
(def :a
  (args)
  (send nil :x
    (int 1)))
```

Reply:
I use `parent` instead of `block_node`.